hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -4442,7 +4459,8 @@ private OmBucketInfo resolveBucketLink(
       UserGroupInformation userGroupInformation,
       InetAddress remoteAddress,
       String hostName,
-      boolean allowDanglingBuckets) throws IOException {
+      boolean allowDanglingBuckets,
+      boolean aclEnabled) throws IOException {
```

Comment:
nit: `aclEnabled` and other parameters are not present in the java-doc comment. Please update it accordingly.